build.gradle

```diff
@@ -80,15 +65,23 @@ task clean(type: Delete) {
 // { ... }`, to avoid issues with importing.
 import com.sun.jna.Platform as DefaultPlatform
 
-// This is the output `git rev-parse HEAD:libs`.  We could discover this automatically, but
-// let's keep things simple and bump this manually for a while.  If and when we grow SNAPSHOT
-// support on maven.mozilla.org, we can invest in a better approach for versioning these
-// prerequisite libraries.
+// If this is `null`, we use libs from the source directory.
+// Check if there are any changes to `libs` since `master`, and if not,
+// use the sha to download the artifacts from taskcluster.
```

Comment:
Do we actually need this part? We don't have any direct external lbs

Reply:
Yeah -- I'd lean on leaving it out -- maybe inserting a comment like:

```
a-s has hooks to load external dependencies here.  This has been removed since `glean-rs` doesn't have any external Kotlin/Java dependencies.
```